packages/datadog-plugin-http/test/integration-test/server.mjs

```diff
@@ -0,0 +1,4 @@
+import 'dd-trace/init.js'
+import * as pluginHelpers from './plugin-helpers.mjs'
+
+pluginHelpers.onMessage(async () => {})
```

Comment:
Is this test piggy-backing on the fact that onMessage is powered by http?

If so, it would probably be better to manually create an http server instead. Doing so makes it more obvious to a future developer what's going on. It also makes it so that we can change how onMessage functions if we want to in the future.